index.css

```diff
@@ -87,6 +95,219 @@ nav {
   }
 
   .social-icon:hover {
-    width: 25px;
+    color: #396df2;
+  }
+
+  .works-sec {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .work-card {
+    display: grid;
+    width: 327px;
+    height: 552px;
+    padding-left: 7%;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    align-items: center;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin-bottom: 88px;
+    background: #fff;
+  }
+
+  .lng-wrk-card {
+    height: 596px;
+  }
+
+  .work-image-cont {
+    width: 295px;
+    height: 220px;
+    background-color: pink;
+    border-radius: 8px;
```

Comment:
@christianbiring1 
thanks alot but where exactly should i add the grid-container .
Could it be the elements are not resizing because of the fixed width declared in px